src/lessons/02_introduction_to_frontend/introduction-to-frontend.slides.html

```diff
@@ -0,0 +1,688 @@
+                  <li class="fragment" data-fragment-index="5"><span>✅</span> In your browser, open the index.html by typing <code style="background: #171758;">file://</code> in your address bar </li>
+                  <li class="fragment" data-fragment-index="7"><span>☑️</span> Now it's time to inspect elements all together! ❤️</li>
+                </ul>
+              </div>
+
+              <preview-window window-title="My first website"
+                              address="file:///C:/Users/redi-user/lesson2/index.html"
+                              icon-class="fa-globe-europe"
+                              class="fragment current-visible"
+                              data-fragment-index="6">
+                <iframe data-src="../01_getting_started/class-resources/first-website.html"></iframe>
+              </preview-window>
+
+              <img data-src="images/vscode.png" class="fragment r-frame current-visible" data-fragment-index="3"/>
+              <img data-src="images/vscode-folder-opened.png" class="fragment r-frame current-visible" data-fragment-index="4"/>
+            </div>
+
+            <aside class="notes">
+              We ain't gonna need a terminal right now but it's good to have it opened for
+            </aside>
+          </section>
+        </section>
+
+        <!--SLIDE 7-->
+        <!--Inspecting elements (demo)-->
+        <section>
+          <!--SLIDE 7.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>Now it's time to inspect elements all together! ❤️</h3>
+            <p class="small-please" style="font-style: italic;">demo being presented</p>
+          </section>
+        </section>
+
+        <!--SLIDE 8-->
+        <!--Other tags-->
+        <section>
+          <!--SLIDE 8.1-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+          </section>
+
+          <!--SLIDE 8.2-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please">Let's learn a few other tags in HTML which will be important in the upcoming lessons and topics</p>
+
+            <aside class="notes">
+              Explain what means "structural", in this case. (Compare with head)
+            </aside>
+          </section>
+
+          <!--SLIDE 8.3-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please">Let's learn a few other tags in HTML which will be important in the upcoming lessons and topics</p>
+
+            <ul class="tags-with-code">
+              <li><code>&lt;div&gt;</code> defines a division or a section in an HTML document.</li>
+              <li><code>&lt;img&gt;</code> used to embed an image in an HTML page. <small style="vertical-align: middle;">(remember: there's not content within img)</small></li>
```

Comment:
I'd use the term "void elements" here, because they are commonly called that elsewhere in for example documentations

```suggestion
              <li><code>&lt;img&gt;</code> used to embed an image in an HTML page. <small style="vertical-align: middle;">(remember: img is a void element - there is no closing tag)</small></li>
```